src/models/SceneModel.js

```diff
@@ -27,11 +27,14 @@ export default class SceneModel extends BaseModel {
     this.displayName = displayName;
     this.clip = clip;
 
-    // handle filename controls
-    if (filename != null) {
```

Comment:
this was the source of the bug that caused `Reset Scene` to break everything.  Apparently when I implemented it originally, I assumed 'filename' was null for everything but video clips.  That's not the case, everything has a filename field, so now we explicitly check for `clipId == 'video'`